pkg/roachprod/cloud/gc.go

```diff
@@ -421,10 +445,18 @@ func GCClusters(l *logger.Logger, cloud *Cloud, dryrun bool) error {
 		if err := destroyResource(dryrun, func() error {
 			return DestroyCluster(l, c)
 		}); err == nil {
-			destroyedClusters = append(destroyedClusters, resourceDescription{
-				Description:      fmt.Sprintf("%s (expiration: %s)", c.Name, c.ExpiresAt().String()),
-				SlackDescription: fmt.Sprintf("`%s` (expiration: %s)", c.Name, slackClusterExpirationDate(c)),
-			})
+			if !c.IsEmptyCluster() {
```

Comment:
Nit: we could lift cloud formatting to avoid always saying `cloud(s)` in the common case of single-cloud clusters. Something like:

```go
clouds := strings.Join(c.Clouds(), ", ")
cloudsDesc := "cloud"
if len(c.Clouds()) > 1 {
    cloudsDesc = "clouds"
}

// ... later ...
// Description: fmt.Sprintf("%s (%s: %s, expiration: %s)", c.Name, cloudsDesc, clouds, c.GCAt().String()),
```

Reply:
Yep, I'll refactor; the duplication was already making me question it.